packages/lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opticks",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
Manual bump as current version is [5.1.1](https://www.npmjs.com/package/opticks)